app/services/participants/action.rb

```diff
@@ -28,7 +28,7 @@ def application
         &.applications
         &.accepted
         &.includes(:course)
-        &.find_by(lead_provider:, course: { identifier: course_identifier })
```

Comment:
not sure about this one, testing locally and getting strange results, let's chat tomorrow

Reply:
tl;dr: this will select any application for a user, which will bring up other validations unnecessarily. For instance a participant has a withdrawn application for course 1 and another active application for course 2. I am trying to defer the second application and pass in the wrong course identifier, it selects the application for course 1 and returns an error for it being withdrawn, which we don't want